tools/topology/m4/muxdemux.m4

```diff
@@ -36,6 +36,15 @@ define(`N_MUXDEMUX', `MUXDEMUX'PIPELINE_ID`.'$1)
 
 dnl W_MUXDEMUX(name, mux/demux, format, periods_sink, periods_source, core, kcontrol_list)
 define(`W_MUXDEMUX',
+`SectionVendorTuples."'N_MUXDEMUX($1)`_tuples_uuid" {'
+`	tokens "sof_comp_tokens"'
+`	tuples."uuid" {'
+`		SOF_TKN_COMP_UUID'		"c4:b2:68:68:14:30:47:0e:a0:89:15:d1:c7:7f:85:1a"
```

Comment:
Could you document the format used for uuid in this token?
If this is just a byte array, is endianess correct? There are uint32, uint32, uint8.
Example from the fw source code:
```c
/* c4b26868-1430-470e-a089-15d1c77f851a */
DECLARE_SOF_UUID("demux", demux_uuid, 0xc4b26868, 0x1430, 0x470e,
		 0xa0, 0x89, 0x15, 0xd1, 0xc7, 0x7f, 0x85, 0x1a);
```
So if it is byte array and little-endian is assumed, it should rather be 68:68:b2:c4:30:14....

Reply:
You are right, thanks for spotting this.